build.gradle

```diff
@@ -89,6 +89,7 @@ ext {
         mockito: 'org.mockito:mockito-core:2.27.0',
         truth: 'androidx.test.ext:truth:1.1.0',
         compiletesting: 'com.google.testing.compile:compile-testing:0.17',
+        compiletestingkt: 'com.github.tschuchortdev:kotlin-compile-testing:1.2.6',
```

Comment:
`kotlin-compile-testing` makes tests flake on my machine in about 50% of times. I worry if that flakes would affect CI. Compiling kotlin files in tests is unnecessary, as we do not use any kotlin specific features in `moxy-compiler`. So I suggest removing `kotlin-compile-testing`